### DISCUSSION TITLE
[HELP]Can i migrate the ~/.chia/mainnet to other places

### DISCUSSION THREAD

**Discussion by johnstonemees** [Chia-Network/chia-blockchain#7717] [Q&A] [answered]
Hi there,i have a problem when i syncing the nodes of chia,the dis of default path was full that i cann't going next step,can i migrate the nodes files to other places?i was tried to modified the ``PWD env in /etc/profile``,but no used.

**Comment by 0xhhhhh** [comment]
You can try the following operations:

```bash
$ mount /dev/sd.. /mnt
$ chia stop -d all
$ mv ~/.chia /mnt/chia
$ ln -s /mnt/chia ~/.chia
$ chia start ...
```

**Comment by johnstonemees** [comment]
Thanks man.

**Comment by H34dl3ss** [comment, marked as answer]
By design you can use the CHIA_ROOT environment variable to set another directory.
https://github.com/Chia-Network/chia-blockchain/wiki/Quick-Start-Guide#install

- stop the chia daemon
- move ~/.chia to the new destination
- set the environment variable by executing
`export CHIA_ROOT=/your/new/destination/.chia/mainnet`
- restart the chia daemon

To make the evironment variable persistent across reboot, insert the line above into your .bashrc
